src/main/config/logback.xml

```diff
@@ -9,6 +9,7 @@
   </appender>
 
   <logger name="org.eclipse.jetty" level="${JETTY_LOG_LEVEL:-info}" />
+  <logger name="org.gaul.shaded.org.eclipse.jetty" level="${JETTY_LOG_LEVEL:-info}" />
```

Comment:
Why have both of these?

Reply:
I don't know enough about how the "-all" jar is run as we don't use it that way so I wasn't able to test this. I assume only the line I added is needed since the classes are now all relocated. This also assumes Jetty have named their loggers after class names, if not then you'd also need the first line.